### PR TITLE
Changes for Istio and OpenServiceMesh

### PR DESCRIPTION
**Pull request by craigbox**
Update the table/s to reflect the [archival of OpenServiceMesh](https://openservicemesh.io/blog/osm-project-update/) and the [graduation of Istio](https://istio.io/blog/2023/istio-graduates-within-cncf/).

### CODE REVIEW

File: _includes/implementations.html [hunk 1]
```diff
@@ -104,7 +92,7 @@ <h2 class="list-teaser__headline">Open Service Mesh (OSM)</h2>
           </div>
           <h2 class="list-teaser__headline">Cilium</h2>
           <p class="list-teaser__text">
-            Cilium existed long before the term Service Mesh even came up, providing security, networking and oservability features efficiently with eBPF instead of sidecars.
+            Cilium existed long before the term Service Mesh even came up, providing security, networking and observability features efficiently with eBPF instead of sidecars.
```
Comment (MarcoSteinke) [inline _includes/implementations.html:95]: Thank you 😄

File: _includes/comparison.html [hunk 2]
```diff
@@ -67,18 +62,16 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://www.consul.io/docs/connect/gateways#ingress-gateways" target="_blank">Envoy</a>. Support for <a href="https://gateway-api.sigs.k8s.io/" target="_blank">Kubernetes Gateway API</a> with Consul API Gateway</td>
         <td>any</td>
         <td>any</td>
-        <td><a href="https://release-v0-11.docs.openservicemesh.io/docs/guides/traffic_management/ingress/" target="_blank">prepared config for Contour</a>, compatible with any other</td>
         <td><a href="https://docs.cilium.io/en/v1.12/gettingstarted/servicemesh/ingress/" target="_blank">Cilium Ingress for TLS & path-based routing features</a>, compatible with any other</td>
       </tr>
       <tr>
           <th scope="row">Governance</th>
-          <td>see <a href="https://github.com/istio/community#istio-community" target="_blank">Istio Community</a> and <a href="https://openusage.org/" target="_blank">Open Usage Commons</a></td>
+          <td>see <a href="https://github.com/istio/community#istio-community" target="_blank">Istio Community</a> and <a href="https://github.com/cncf/foundation/blob/master/charter.md" target="_blank" >CNCF Charter</a></td>
```
Comment (MarcoSteinke) [inline _includes/comparison.html:69]: 👍🏻

**Comment by craigbox** [comment]
Who can merge this? @MarcoSteinke

**Comment by MarcoSteinke** [comment]
I will merge this later on, I first had to think about how to change the presentation of OSM to make the archival clear to all visitors.

**Pull request merged**